packages/onenoteanalyzer.vm/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,27 @@
+$ErrorActionPreference = 'Stop'
+Import-Module vm.common -Force -DisableNameChecking
+
+try {
+
+    $toolName = 'OneNoteAnalyzer'
+    $category = 'Office'
+
+    $toolDir = Join-Path ${Env:RAW_TOOLS_DIR} $toolName
+    $zipUrl = 'https://github.com/knight0x07/OneNoteAnalyzer/releases/download/OneNoteAnalyzer/OneNoteAnalyzer-withPass.zip'
+    $zipSha256 = 'a8b04313cf73360b828624a423435b2f2b4652a16c421170b2400966e6ac77cc'
+
+    $packageArgs = @{
+        packageName    = ${Env:ChocolateyPackageName}
+        unzipLocation  = $toolDir
+        url            = $zipUrl
+        checksum       = $zipSha256
+        checksumType   = 'sha256'
+    }
+    Install-ChocolateyZipPackage @packageArgs
+    VM-Assert-Path $toolDir
+
+    $executablePath = Join-Path $toolDir "OneNoteAnalyzer-withPass\OneNoteAnalyzer_withPass.exe" -Resolve
+    VM-Install-Shortcut -toolName $toolName -category $category -executablePath $executablePath -consoleApp $true
```

Comment:
I think we should provide `-arguments "--help"` (note it was removed by default in https://github.com/mandiant/VM-Packages/pull/762) to avoid displaying the error message
![Screenshot 2023-12-21 at 12 51 56](https://github.com/mandiant/VM-Packages/assets/16052290/8385948c-a5a8-4c68-93c4-7761ccf5f525)

Reply:
Fixed.